see/image_providers/os_glance.py

```diff
@@ -18,15 +18,17 @@
 by name or UUID; if name is requested the latest matching image is retrieved.
 
 provider_parameters:
-    path (str):     Absolute path where to download the image. If target_path
-                    is an existing file, it will be overwritten if the image
-                    is newer. Otherwise target_path is understood to be a
-                    directory and the image's UUID will be used as filename.
-    os_auth (dict): A dictionary with OpenStack authentication parameters as
-                    needed by OpenStack's Keystone client.
-    session (dict): A dictionary with OpenStack Session parameters. Allows
-                    authentication to Keystone over TLS.
-
+    path (str):          Absolute path where to download the image.
+                         If target_path is an existing file, it will be
+                         overwritten if the image is newer. Otherwise
+                         target_path is understood to be a directory and the
+                         image's UUID will be used as filename.
+    os_auth (dict):      A dictionary with OpenStack authentication parameters
+                         as needed by OpenStack's Keystone client.
+    session (dict):      A dictionary with OpenStack Session parameters. Allows
+                         authentication to Keystone over TLS.
+    libvirt_pool (dict): An optional dictionary with backing libvirt pool
```

Comment:
Where can I find the structure of this dict?

Reply:
You'll have to dig out my brain! 😁
At the moment we're just looking for two keys, 'hypervisor' and 'name'